docs/src/languages/php/redis.md

```diff
@@ -6,103 +6,49 @@ weight: 7
-    # Tell PHP to enable the extension.
-    echo "Enabling PhpRedis extension."
-    echo -e "\nextension=${PLATFORM_APP_DIR}/redis.so" >> $PLATFORM_APP_DIR/php.ini
-}
-
-copy_lib() {
-    # Copy the compiled library to the application directory.
-    echo "Installing PhpRedis extension."
-    cp $PLATFORM_CACHE_DIR/phpredis/modules/redis.so $PLATFORM_APP_DIR
-}
-
-checkout_version () {
-    # Check out the specific Git tag that we want to build.
-    git checkout "$1"
-}
-
-ensure_source() {
-    # Ensure that the extension source code is available and up to date.
-    if [ -d "phpredis" ]; then
-        cd phpredis || exit 1;
-        git fetch --all --prune
-    else
-        git clone https://github.com/phpredis/phpredis.git
-        cd phpredis || exit 1;
-    fi
-}
-
-compile_source() {
-    # Compile the extension.
-    phpize
-    ./configure
-    make
-}
-
-ensure_environment() {
-    # If not running in a Platform.sh build environment, do nothing.
-    if [ -z "${PLATFORM_CACHE_DIR}" ]; then
-        echo "Not running in a Platform.sh build environment.  Aborting Redis installation."
-        exit 0;
-    fi
-}
-
-ensure_arguments() {
-    # If no version was specified, don't try to guess.
-    if [ -z $1 ]; then
-        echo "No version of the PhpRedis extension specified.  You must specify a tagged version on the command line."
-        exit 1;
-    fi
-}
-
-ensure_environment
-ensure_arguments "$1"
-run "$1"
-```
-
-2. Invoke that script from your build hook, specifying a version.  Any tagged version of the library is acceptable:
+A [script](https://github.com/platformsh/snippets/blob/main/src/install-phpredis.sh) is provided to ease the installation of a customer version of PhpRedis.  Invoke that script from your build hook, specifying a version.  Any tagged version of the library is acceptable:
 
-```yaml
+```yaml {location=".platform.app.yaml"}
 hooks:
     build: |
         set -e
-        bash install-redis.sh 5.1.1
+        # Install PhpRedis v5.1.1:
+        curl -fsS https://raw.githubusercontent.com/platformsh/snippets/main/src/install-phpredis.sh | { bash /dev/fd/3 5.1.1 ; } 3<&0
 ```
 
-3. If you ever wish to change the version of PhpRedis you are using, update the build hook and clear the build cache: `platform project:clear-build-cache`.  The new version will *not* be used until you clear the build cache.
+## Install Relay
+
+Relay is a [Redis](../../add-services/redis.md) client like [PhpRedis](https://github.com/phpredis/phpredis) and [PhpRedis](https://github.com/predis/predis). It intents to be a drop-in replacement of those libraries.
+
+That PHP extension is also a shared in-memory cache like APCu. All retrieved keys are held in the PHP master process’ memory, which is shared across all FPM workers.
+
+That means if the FPM Worker #1 fetches the `users:count` key from Redis, the other FPM Workers #2, #3, … will instantaneously retrieve that key from Relay, without having to communicate with Redis.
```

Comment:
:hammer_and_wrench: We prefer to avoid the future tense: https://github.com/platformsh/platformsh-docs/blob/main/contributing/content-style.md#use-the-present-tense

```suggestion
That means if the FPM Worker #1 fetches the `users:count` key from Redis,
all other FPM workers instantaneously retrieve that key from Relay without having to communicate with Redis.
```